src/OpenTK/Math/Vector2d.cs

```diff
@@ -627,6 +631,118 @@ public static void Max(ref Vector2d a, ref Vector2d b, out Vector2d result)
 
         #endregion
 
+        #region ComponentMin
+
+        /// <summary>
+        /// Calculate the component-wise minimum of two vectors
+        /// </summary>
+        /// <param name="a">First operand</param>
+        /// <param name="b">Second operand</param>
+        /// <returns>The component-wise minimum</returns>
+        public static Vector2d ComponentMin(Vector2d a, Vector2d b)
```

Comment:
Compounds isn't that clear to the average person I'd venture. We ideally want to write in fairly simple, non-mathematical english here, as a lot of game developers and library users are hobbyists.